cypress/integration/admin_account_test/report_page_layout_test.js

```diff
@@ -0,0 +1,100 @@
+describe("Report Page Layout Test", function(){
+    beforeEach(function(){
+        cy.openLoginDialogue();
+		cy.get(".login-form").should("contain", "Login to BioTestMine");
+		cy.get("input#email").type("test_user@mail_account");
+		cy.get("input[type='password']").type("secret");
+        cy.intercept('POST', '/api/auth/login').as('login');
+		cy.get(".login-form")
+			.find("button")
+			.contains('Login')
+		 	.click();
+        cy.wait('@login');
+        cy.get(".logon.dropdown.success").should("exist").click();
+        cy.get(".logon.dropdown.success").should("contain", "test_user@mail_account"); //flaky
+        cy.visit("/biotestmine/admin");
+        cy.contains("Report page layout").parent().should("have.class","well").as("reportPageLayout");
+    });
+
+    it("can add a category containing classes",function(){
+        cy.get("@reportPageLayout").within(() => {
+            cy.get("Select").select("MRNA");
+            cy.get("select").should("have.value","MRNA");
+            cy.get("input").should("exist").type("Interactions{enter}",{delay:100});
+            cy.get("li").contains("Interactions").should("exist").click(); //Flaky
+            cy.get('.css-1hwfws3').click().type("Exons{enter}",{delay:100},{force:true});
+            cy.get("ul").filter(".classes").find(".btn-fab").first().click();  //Flaky
+            cy.contains("Save changes").click();
+            cy.get(".success").should("include.text","Successfully saved changes to report page layout.");  
+        })
+    })
+
+    it("can add description to classes", function(){
+        cy.get("@reportPageLayout").within(() => {
+            cy.get("Select").select("MRNA");
+            cy.get("select").should("have.value","MRNA");
+            cy.get("li").contains("Interactions").should("exist").click();
+            cy.get("li").contains("Exons").should("exist").click();
+            cy.get(".btn-group-sm").children().first().click();
+
+            cy.get(".classes").within(() => {
+                cy.get("textarea").clear().type("Coding regions within this gene{enter}",{delay:100});
+                cy.contains("Save").click();
+            })
+
+            cy.contains("Save changes").click();
+            cy.get(".success").should("include.text","Successfully saved changes to report page layout.");  
+        })
+    })
+
+    it("can edit the visibility of the description of a class", function(){
```

Comment:
This test name is misleading - the "eye" button is used to collapse the content by default (table or visualization on the report page) so that the user has to click to expand if they wish to view it. Can you change it to "can make a class show as collapsed"?